### DISCUSSION TITLE
v1.0.2

### DISCUSSION THREAD

**Discussion by loks666** [loks666/get_jobs#6] [Announcements]
- 更新了若干问题
- 更新了readme文档
- 添加了智联招聘投递
- 添加了全局cookie支持

**Full Changelog**: https://github.com/loks666/get_jobs/compare/v1.0.1...v1.0.2

<hr /><em>This discussion was created from the release <a href='https://github.com/loks666/get_jobs/releases/tag/v1.0.2'>v1.0.2</a>.</em>

**Comment by lixiang419** [comment]
大佬好棒~~

**Reply by ZhYaL** [reply]
现在关闭了上传许愿墙权限了吗?呜呜呜,我也想成为贡献者,呜呜呜哭哭

**Comment by GeneralFly666** [comment]
同意楼上

**Comment by zt8960** [comment]
我觉得不应该局限于招聘平台，因为这种地方工作质量真的不能保证。

**Reply by loks666** [reply]
那你有什么好的解决办法吗？

**Comment by afd9986-tech** [comment]
谢谢大佬

**Comment by TK88101** [comment]
想問一下 該項目 支持的是中國大陸的就業app，那有沒有可能把日本的就業app 也覆蓋上呢
